app/services/cosmos/cosmos.utils.ts

```diff
@@ -50,3 +69,129 @@ export function calculateUnbondingDelegationBalanceFromResultEntries(entries: Co
 export function validateAccountAddress(address: string) {
   return /^cosmos1[ac-hj-np-z02-9]{38}$/.test(address)
 }
+
+export function convertDelegationDelegatorReward
+  (reward: DelegationDelegatorReward): CosmosValidatorReward {
+  const { validatorAddress, reward: rewardInput } = reward
+  const coins = rewardInput.map(coin => coin as CosmosCoinResult)
+  return { validator_address: validatorAddress, reward: coins }
+}
+
+function convertDescription
+  (description: Description = {} as Description) {
+  const {
+    moniker = '',
+    identity = '',
+    website = '',
+    details = ''
+  } = description;
+  return { moniker, identity, website, details }
+}
```

Comment:
I mean what exactly does this function convert? converting Description to Description?

Reply:
Reference [here](https://github.com/confio/cosmjs-types/blob/main/src/cosmos/staking/v1beta1/staking.ts#L102).
The original `Description` type has another field `securityContact `.
I think this converter filters this field here. 
Besides, it provides a default value for `Description`'s type fields since [description is optional](https://github.com/confio/cosmjs-types/blob/main/src/cosmos/staking/v1beta1/staking.ts#L131).